docs/concepts/manifest-builds.md

```diff
@@ -92,6 +93,7 @@ ripgrep.pkg-path = "ripgrep"
 
 [build.hello-pkg]
 command = '''
+  mkdir -p $out/bin
```

Comment:
nitpick(non-blocking): See previous comment. It's a good idea to keep the examples consistent.